src/test/java/com/wikia/webdriver/elements/communities/desktop/pages/SpamWikiReviewPage.java

```diff
@@ -0,0 +1,96 @@
+package com.wikia.webdriver.elements.communities.desktop.pages;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.Collections;
+import java.util.List;
+
+public class SpamWikiReviewPage extends BasePageObject {
+
+  private static final String SPAM_WIKI_REVIEW_SERVICE_NAME = "spam-wiki-review";
+  private static final String VIEW_WIKIS_SUBPAGE = "/wiki";
+
+  public static enum LANGUAGE_CODE {
+    all, en, ar, de, es, fi, fr, id, hi, hu, it, ja, ka, kn, ko, nl, pt, pl, ru, uk, vi, zh, other
+  }
+
+  public static String QUESTIONABLE_BUTTON_TEXT = "Questionable";
+  public static String SPAM_BUTTON_TEXT = "Spam";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make SPAM_BUTTON_TEXT a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this "public static SPAM_BUTTON_TEXT" field final [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1444)